plugin.php

```diff
@@ -306,7 +306,7 @@ function rest_validate_request_arg( $value, $request, $param ) {
 		if ( isset( $args['format'] ) ) {
 			switch ( $args['format'] ) {
 				case 'date-time' :
-					if ( ! rest_parse_date( $value ) ) {
+					if ( $value && ! rest_parse_date( $value ) ) {
```

Comment:
@danielbachhuber I'd be in favour of treating `null` as the value not being included. Maybe we should do an `array_filter` to exclude null before we validate?

Also, would this break things if we handled it that way?